### PR TITLE
Add the Eraser to the available tools

### PR DESCRIPTION
**Pull request by dlwire**
Adding the Eraser tool to the list of supported tools.

We could also add this to the toolbar easily enough but here we go for now.

https://github.com/trustsitka/sitka/issues/3403

### CODE REVIEW

File: src/setupTools.js [hunk 1]
```diff
@@ -217,6 +217,15 @@ export default function setupTools(store) {
     { name: 'ZoomTouchPinch' },
     { name: 'StackScrollMouseWheel' },
     { name: 'StackScrollMultiTouch' },
+    {
+      name: 'Eraser',
+      mouseButtonMasks: [1],
+      props: {
```
Comment (swederik) [inline src/setupTools.js]: I don't think the Eraser tool should have these options

Comment (dlwire) [reply src/setupTools.js]: You're absolutely right. Looks like I added this while I was trying to resolve a different issue that turned out to be caused by something in toolbarModule.js, which isn't included in this review. I'll get a fix.

Thanks Erik!

Comment (dlwire) [reply src/setupTools.js]: I also removed the mouseButtonMasks this morning. Turns out that wasn't need either.

**Comment by ranasrule** [comment]
its still not visible in the toolbar. am I missing something?

**Comment by swederik** [comment]
Yes it hasn't been added to the toolbarModule: https://github.com/OHIF/Viewers/blob/master/extensions/ohif-cornerstone-extension/src/toolbarModule.js#L28

**Pull request merged**